app/gui/src/project-view/components/shared/AgGridTableView.vue

```diff
@@ -123,6 +123,7 @@ const props = defineProps<{
   datasource?: IServerSideDatasource
   rowCount?: number
   isServerSideModel?: boolean
+  nodeType?: string
```

Comment:
`AgGridTableView` is a general-purpose component for displaying tables in our app, and I don't know what "nodeType" means here. Shouldn't it be handled in TableVisualization?